src/bubble/bubbles.clj

```diff
@@ -40,12 +41,16 @@
     (println (bubble-members param-id))
     (views/base-view
      [[:h1 (:bubbles/name bubble)]
+      [:h2 (str "Join bubble here: localhost:3000/bubble/" param-id "/join")]
```

Comment:
Might be nice to include a link to a pre-filled SMS message. The following should work:

```clojure
(let [join-link (str "http://localhost:3000/bubble/" param-id "/join")]
  [:a {:href (str "sms:?&=" (ring.util.codec/form-encode {:body (str "Join my bubble: " join-link}))} (str "Bubble join link: " join-link)]
```